Contributors.txt

```diff
@@ -18,13 +18,13 @@ Developers:
 	Thierry Bordaz
 	Sumit Bose
 	François Cami
+	Petr Čech
```

Comment:
Seems like the lexicographic ordering isn't quite right, `Č` should come after `C`